### DISCUSSION TITLE
Bypassing stadia country verification

### DISCUSSION THREAD

**Discussion by AntonIXO** [ChristopherKlay/StadiaEnhanced#123] [Ideas & Requests]
Hi, I am living in Russia and Stadia don't give to play in my country.
![image](https://user-images.githubusercontent.com/25568560/114624043-bd5f7f80-9cb8-11eb-913b-e37275ed3dd2.png)
Maybe you remove part of the code responsible for country checking?
Thanks

**Comment by ChristopherKlay** [comment]
This most likely requires a VPN, due to not having any access to the (server sided) parts that check your connection.
It could also create legal issues i would like to avoid, so i have to decline this request.

I can help you out with a solution however; You can use a VPN (even a free/limited one in this case) to start the game and once it's started and the session is created (i.e. the game is fully running) you can turn the VPN off - no need to worry about added latency or such things due to the VPN. The check for country restrictions is - so far - only during the creation of a session.
